conda_build/skeletons/cpan.py

```diff
@@ -180,8 +177,24 @@ def __enter__(self):
             return dst
 
 
-def loose_version(ver):
-    return str(parse_version(str(ver)))
+def LooseVersion(version):
+    """
+    Wrap distutils.version.LooseVersion, coercing passed version into str
+    and preserving None passed in
+
+    metacpan is sometimes bad about returning strings or numbers for versions
+    see https://github.com/metacpan/metacpan-api/issues/789
+
+    we also sometimes get version numbers prefixed with 'v' and that causes
+    TypeError about comparing ints against strings
+
+    :param version: version string or number
+    :return: distutils.version.LooseVersion(str(version).lstrip('v'))
+    """
+    if version is None:
```

Comment:
This block could be just:

```
if version:
    return  distutils.version.LooseVersion(str(version).lstrip('v'))
```

because python's implicit return type is None anyway.

Reply:
@msarahan good point.  I'll change this.

Reply:
Resolved in https://github.com/conda/conda-build/pull/3671/commits/fa7f928e22ab8374de923e427f3f0a46790e6b6d